dev.exs

```diff
@@ -28,6 +28,88 @@ Application.put_env(:phoenix_live_dashboard, DemoWeb.Endpoint,
   ]
 )
 
+defmodule DemoWeb.History do
+  use GenServer
+  alias Phoenix.LiveDashboard.TelemetryListener
+
+  @history_buffer_size 50
+
+  def data(metric) do
+    GenServer.call(__MODULE__, {:data, metric})
+  end
+
+  def start_link(args) do
+    GenServer.start_link(__MODULE__, args, name: __MODULE__)
+  end
+
+  def init(metrics) do
```

Comment:
You should call `Process.flag(:trap_exit, true)` and remove the handlers on `terminate`. See the example in the telemetry_metrics guide.